bundle/bundle.go

```diff
@@ -50,6 +51,9 @@ type Bundle struct {
 	clientOnce sync.Once
 	client     *databricks.WorkspaceClient
 
+	// Files that are synced to the workspace.file_path
+	SyncedFiles []fileset.File
```

Comment:
That they are synchronized is implied, IMO. We can use just `Files`.

Reply:
Done